internal/pkg/cli/init.go

```diff
@@ -126,8 +130,13 @@ func newInitOpts(vars initVars) (*initOpts, error) {
 		initEnvVars: initEnvVars{
 			GlobalOpts:   NewGlobalOpts(),
 			Name:         defaultEnvironmentName,
-			Profile:      vars.profile,
 			IsProduction: false,
+
+			TempCreds: tempCredsVars{
```

Comment:
Are this temp cred only configurable via `copilot init`? (Can users specify them using flags when doing `env init`?)

Reply:
Jes, next PR will add the flags so that users can configure them via flags in `env init`.